lib/inbox.rb

```diff
@@ -347,15 +347,13 @@ def delta_stream(cursor, exclude_types=[], timeout=0, expanded_view=false)
         end
       end
 
-      EventMachine.run do
-        http = EventMachine::HttpRequest.new(path, :connect_timeout => 0, :inactivity_timeout => timeout).get(:keepalive => true)
-        http.stream do |chunk|
-          parser << chunk
-        end
-        http.errback do
```

Comment:
Ah, I meant the `http.errback` functionality to catch errors - I removed, then re-added it.